### DISCUSSION TITLE
why is container unhealthy, how can I debug the problem

### DISCUSSION THREAD

**Discussion by karazonanas** [shyim/shopware#54] [Q&A]
my container based on  image: shyim/shopware:6.4.15.2 got sometimes (suddenly) unhealthy

I can't recognize a "pattern" when this is happening. Even without restarting the container it suddenly get unhealthy and later then healthy again without me doing anything.

Today I got this failure `Health check exceeded timeout`

so any Idea what can I do about that? is there anyway to debug what's happening during the health check?

**Comment by shyim** [comment]
It can be that the migration etc. are running until its completely healthy. The health check does curl at localhost/admin and waits until its reachable
